doc/plugins/response-rewrite.md

```diff
@@ -116,3 +117,11 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f1
 ```
 
 The `response rewrite` plugin has been disabled now. It works for other plugins.
+
+## Attention
+
+`ngx.exit` will interrupt the execution of the current request and return status code to Nginx.
+
+![](https://cdn.jsdelivr.net/gh/Miss-you/img/picgo/20201113010623.png)
+
+However, if you execute `ngx.exit` during the access phase, it only interrupts the request processing phase, and the response phase will still process it, i.e. if you configure the `response-rerite` plugin, it will force overwriting of your response information (e.g. response code).
```

Comment:
Please check the plugin name

Reply:
Not sure if response code is the same as response status code.

Reply:
done


> Please check the plugin name

Reply:
> response status code
response code ->
response status code